README.rst

```diff
@@ -23,27 +23,45 @@ Installation
 
 PennyLane-Lightning requires Python version 3.6 and above. It can be installed using ``pip``:
 
-.. code-block:: bash
+.. code-block:: console
 
-    pip install pennylane-lightning
+    $ pip install pennylane-lightning
 
 Alternatively, to build PennyLane-Lightning from source you can run
 
-.. code-block:: bash
+.. code-block:: console
 
-    git clone https://github.com/XanaduAI/pennylane-lightning.git
-    cd pennylane-lightning
-    pip install -e .
+    $ git clone https://github.com/XanaduAI/pennylane-lightning.git
+    $ cd pennylane-lightning
+    $ pip install -e .
```

Comment:
Though not tightly connected, thought could be added in this PR. An idea that could be nice to be included in the docs (feel free to rephrase).
```suggestion
    $ pip install -e .

    Subsequent calls to `pip install -e .` will use cached binaries stored in the `build` folder. Use the `python setup.py clean --all` command to clean cached files and to re-build the binaries with `pip install -e .`.
```

Reply:
Thanks, have added!